tgui/packages/tgui/interfaces/common/mathUtils.ts

```diff
@@ -0,0 +1,5 @@
+export const randInt = (a: number, b: number) => {
+  const min = b > a ? a : b;
+  const max = b > a ? b : a;
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+};
```

Comment:
If we're making a common util out of this, I would formalize it as:
```suggestion
export const randInt = (min: number, max: number) => {
  return Math.floor(Math.random() * (max - min + 1)) + min;
};
```
If there's ever a case that an interface doesn't know which number is smaller, it can deal with that logic (or you could make a second function to handle it).